blogposts/announcing-sourcegraph-3.10.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Sourcegraph 3.10: Improved search autocompletion, native GitLab integration, and find-and-replace automation campaigns"
+author: Christina Forney
+publishDate: 2019-11-20T10:00-07:00
+tags: [blog]
+slug: sourcegraph-3.10
+heroImage: https://about.sourcegraph.com/sourcegraph-mark.png
+published: true
+---
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most advanced technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
```

Comment:
I would invert the order of these two sentences ("Sourcegraph does X, it is popular" instead of "Sourcegraph is popular, it does X")

Reply:
I'm going to re-write this section for the releases going forward. Thanks for the suggestion!